src/test/java/subway/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,175 @@
+package subway;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import subway.api.LineTestApi;
+import subway.api.SectionTestApi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("지하철 구간 관리")
+@Sql(scripts = {"classpath:sql/truncate.sql", "classpath:sql/setupSectionTest.sql"})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SectionAcceptanceTest {
+
+	LineTestApi lineApi = new LineTestApi();
+	SectionTestApi sectionApi = new SectionTestApi();
+
+	/** 구간 등록 기능 **/
+	/**
+	 * When 지하철 구간을 등록하면
+	 * Then 지하철 노선 조회 시 등록한 구간을 찾을 수 있다.
+	 * */
+	@DisplayName("지하철 노선에 구간을 등록")
+	@Test
+	void createSection() {
+		// when
+		long registeredUpStationId = 2L;
+		long newDownStationId = 4L;
+		long lineId = lineApi.createLine("신분당선", "bg-red-600", 1L, registeredUpStationId, 10).jsonPath().getLong("id");
+
+		sectionApi.createSection(lineId, newDownStationId, registeredUpStationId, 10);
+
+		// then
+		ExtractableResponse<Response> showLineResponse = lineApi.showLineById(lineId);
+		List<Long> stationIds = showLineResponse.jsonPath().getList("stations.id", Long.class);
```

Comment:
```suggestion
		List<Long> stationIds = showLineResponse;
```
응답에 대해서 stationIds만 주면 더 간결해질 것 같아요! 

이 부분에 대해서 어떻게 생각하시나요!? 😊